lib/slacker/application.rb

```diff
@@ -140,8 +157,27 @@ def configure_db
       end
     end
 
+    def configure_db_tiny_tds
+      begin
+        @database = TinyTds::Client.new :username => @configuration.db_user, :password => @configuration.db_password, 
+                    :host => @configuration.db_server, :database => @configuration.db_name, :port => 1433
```

Comment:
I believe here you wanted to set the port to @configuration.db_port as opposed to hard-coding to 1433.